charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.26.0
-version: 14.1.2
+version: 14.1.3
```

Comment:
Chart version needs to bump to latest.

Reply:
Hi, I've reopened PR here: https://github.com/prometheus-community/helm-charts/pull/1084

Sorry for that 😅